samples/robotframework-browser-library/tests/homepage-tests.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library   Browser
+
+*** Test Cases ***
+Website Launch Test
+    New Page    https://playwright.dev
+    Get Text    h1    contains    Playwright
+
+Home Page Test
+    New Page    https://www.testrail.com/
+    Click       xpath=//*[@id="home_hero"]/div[3]/div[2]/div[2]/div[2]/a
```

Comment:
Please try to avoid long and meaningless xpath expressions in tutorials. Use text and array notation if needed.

Reply:
its done